app/data/AtomListStore.scala

```diff
@@ -1,30 +1,38 @@
 package data
 
-import java.time.Instant
 import com.gu.atom.data.PreviewDynamoDataStore
 import com.gu.media.CapiAccess
-import com.gu.media.model.Platform.Url
-import com.gu.media.model.{ContentChangeDetails, Image, MediaAtom, SelfHostedAsset, VideoAsset, YouTubeAsset}
+import com.gu.media.model.{ContentChangeDetails, Image, MediaAtom}
 import com.gu.media.util.TestFilters
 import model.commands.CommandExceptions.AtomDataStoreError
 import model.{MediaAtomList, MediaAtomSummary}
 import play.api.libs.json.{JsArray, JsValue}
 
 trait AtomListStore {
-  def getAtoms(search: Option[String], limit: Option[Int], shouldUseCreatedDateForSort: Boolean, shouldFilterForSelfHosted: Boolean): MediaAtomList
+  def getAtoms(search: Option[String], limit: Option[Int], shouldUseCreatedDateForSort: Boolean, mediaPlatform: Option[String]): MediaAtomList
 }
 
 class CapiBackedAtomListStore(capi: CapiAccess) extends AtomListStore {
-  override def getAtoms(search: Option[String], limit: Option[Int], shouldUseCreatedDateForSort: Boolean, shouldFilterForSelfHosted: Boolean): MediaAtomList = {
+  override def getAtoms(search: Option[String], limit: Option[Int], shouldUseCreatedDateForSort: Boolean, mediaPlatform: Option[String]): MediaAtomList = {
     // CAPI max page size is 200
     val cappedLimit: Option[Int] = limit.map(Math.min(200, _))
 
+    val dateSorter = shouldUseCreatedDateForSort match {
+      case true => Map("order-date" -> "first-publication")
+      case false => Map.empty
+    }
```

Comment:
Could be an if statement. I'm using pattern matching to achieve consistency with the filter below - makes it a little easier to read.